src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -70,6 +70,11 @@ std::string getEntities
   string    idPattern  = ciP->uriParam["idPattern"];
   string    q          = ciP->uriParam["q"];
 
+  if (ciP->uriParam["options"] == "count")
```

Comment:
In that case, I think reference to #958  inside #1041 can be misleading.  I would change the text for this self PR also